src/containers/pages/FocusInvestigation/map/active/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -347,6 +358,13 @@ exports[`containers/pages/FocusInvestigation/activeMap renders correct detail vi
 </div>
 `;
 
+exports[`containers/pages/FocusInvestigation/activeMap works correctly with Redux store when there is just one goal: GisidaLite layers 1`] = `
+Array [
+  "main-plan-layer-313056f2-2a21-454e-9f88-63207acb00d4",
```

Comment:
This test proves that the `RACD_register_families-point` layer is now passed on to GisidaLite.